ospf6d/ospf6_message.c

```diff
@@ -1696,9 +1701,13 @@ static void ospf6_send(struct in6_addr *src, struct in6_addr *dst,
 	}
 
 	/* send message */
-	len = ospf6_sendmsg(src, dst, &oi->interface->ifindex, iovector);
-	if (len != ntohs(oh->length))
-		flog_err(EC_LIB_DEVELOPMENT, "Could not send entire message");
+	if (oi->area->ospf6->fd > 0) {
```

Comment:
Nit: I think the correct check here is: `if (oi->area->ospf6->fd != -1)`. It is possible to open a file descriptor with number `0` especially after closing stdin (its unlikely).

Reply:
Fixed.